test/functional/page_objects/visual_builder_page.js

```diff
@@ -59,6 +60,17 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }) {
     async enterMarkdown(markdown) {
       const prevRenderingCount = await PageObjects.visualize.getVisualizationRenderingCount();
       const input = await find.byCssSelector('.tvbMarkdownEditor__editor textarea');
+      await this.clearMarkdown();
+      await input.type(markdown);
+      await PageObjects.common.sleep(500);
```

Comment:
I don't think we need `sleep` here. `waitForRenderingCount` for checking condition is doing `retry` up to ~40 sec